.github/workflows/main.yaml

```diff
@@ -0,0 +1,89 @@
+name: NPM
+on:
+    pull_request:
+    push:
```

Comment:
this should get triggered on release as well right?

Reply:
Release happens on merge to feature branch and master. This part is handled by semantic release. 
If it is master the increment version will 1.0.1. If it is feature branch the version will be 1.0.1-feature-name.1.